Feature: configurable modal link (#150)
afterpay/sdk-android: README.md

Given the above, when clicking the more info "link", the modal that opens will be white in the current locale as set in configuration.

###### Modal Link Style Options
A value that can be set on `moreInfoOptions` when initialised. Setting this is optional and is of type `ModalLinkStyle`.

Available values are `CircledInfoIcon`, `MoreInfoText`, `LearnMoreText`, `CircledQuestionIcon`, `CircledLogo`, `Custom`, `None`.

> `CircledInfoIcon` is the default & `None` will remove the link all together.

Reviewer: ```suggestion
`CircledInfoIcon` is the default & `None` will remove the link altogether.
```

> When using `Custom` the `setContent` (takes a single paramater of type `SpannableStringBuilder`) method should be called first (see second example below).

Reviewer: ```suggestion
When using `Custom` the `setContent` (takes a single parameter of type `SpannableStringBuilder`) method should be called first (see second example below).
```

```kotlin
val afterpayBreakdown = view.findViewById<AfterpayPriceBreakdown>(R.id.afterpayPriceBreakdown)
afterpayBreakdown.moreInfoOptions = AfterpayMoreInfoOptions(
    modalLinkStyle  = AfterpayModalLinkStyle.CircledInfoIcon
)
```

Given the above, the price breakdown modal link will be a circle containing a question mark.

```kotlin
val afterpayBreakdown = view.findViewById<AfterpayPriceBreakdown>(R.id.afterpayPriceBreakdown)
val content = SpannableStringBuilder().apply {
    append("Click ")
    append("Here ", StyleSpan(Typeface.BOLD), Spannable.SPAN_EXCLUSIVE_EXCLUSIVE)
}
AfterpayModalLinkStyle.Custom.setContent(content)
afterpayBreakdown.moreInfoOptions = AfterpayMoreInfoOptions(
    modalLinkStyle  = AfterpayModalLinkStyle.Custom
)
```

> Given the above, the price breakdown modal link will be display "Click Here".

Reviewer: ```suggestion
Given the above, the price breakdown modal link will display "Click Here".
```
